config.xmi

```diff
@@ -157,6 +157,11 @@
     <title text="Schutzgebiete"/>
     <objectType>ps_4.0:ProtectedSite</objectType>
   </layerConfig>
+  <layerConfig name="PS_ProtectedSite" registryId="http://inspire.ec.europa.eu/layer/PS.ProtectedSite" tags="//@tags.0 //@tags.2" layerName="PS.ProtectedSite" styleConfig="//@styleConfig.106">
```

Comment:
As the layer in the config.style file only has one tag, do 2 tags need to be listed here?

Reply:
@stempler Do you know what the tags are referring to?